civihr_default_theme/scss/components/_manager_dashboard.scss

```diff
@@ -0,0 +1,52 @@
+.chr_manager_dashboard__header {
+  margin: 0;
+
+  .chr_custom-select {
+    margin-right: 20px;
+    padding: 0;
+    width: 250px;
+  }
+
+  .panel-title {
+    margin-top: 5px;
+    display: inline-block;
+  }
+
+  > div {
```

Comment:
Updated the Markup so that we dont need to define the class here, instead used bootstrap class `pull-right`